client/jetpack-connect/jetpack-new-site/index.jsx

```diff
@@ -72,13 +72,13 @@ class JetpackNewSite extends Component {
 						<Card className="jetpack-new-site__wpcom-site">
 							<WordPressLogo />
 							<h3 className="jetpack-new-site__card-title">
-								{ this.props.translate( 'Create a new shiny WordPress.com site' ) }
+								{ this.props.translate( 'Create a shiny new WordPress.com site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Create a new shiny WordPress.com site:' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/new-site-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).